pkg/apis/litmuschaos/v1alpha1/chaosengine_types.go

```diff
@@ -32,7 +32,7 @@ type ChaosEngineSpec struct {
 	AnnotationCheck string `json:"annotationCheck,omitempty"`
 	//ChaosServiceAccount is the SvcAcc specified for chaos runner pods
 	ChaosServiceAccount string `json:"chaosServiceAccount"`
-	//Components contains the image of runnner and monitor pod
+	//Components contains the image of runnner
```

Comment:
includes imagePullPolicy also with recent additions, right?

Reply:
All these changes are for runner, Monitor only had image

Reply:
Correct. Above comment says "Components contains the image of runner" I guess we can enhance the comment with these details also..